2022/08/23 18:57:11 [error] 30#30: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/2.0", upstream: "fastcgi://172.19.0.5:9000", host: "172.17.0.1"
2022/08/23 19:00:04 [error] 30#30: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/2.0", upstream: "fastcgi://172.19.0.5:9000", host: "172.17.0.1"
2022/08/23 19:02:59 [error] 30#30: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/2.0", upstream: "fastcgi://172.19.0.5:9000", host: "172.17.0.1"
